src/mazzaroth.js

```diff
@@ -440,29 +440,33 @@ subCmd.action(function (val, options) {
   process.stdin.on('data', process.exit.bind(process, 0))
 })
 
-const deployCmd = program.command('deploy <val>')
+const deployCmd = program.command('deploy [input]')
 const deployCmdDescription = `
 Helper for deploying a contract to a mazzaroth network. Takes a json config file,
 a sample config file can be found at
 https://github.com/kochavalabs/full-contract-example/blob/master/deploy.json
 
 Examples:
   mazzaroth-cli deploy ./deploy.json
+  echo '{}' | mazzaroth-cli deploy ./deploy.json
```

Comment:
Remove the ./deploy.json from this example.

Reply:
Opppp, good call